### DISCUSSION TITLE
Keep filter sidebar open on big screens

### DISCUSSION THREAD

**Discussion by noxhirsch** [home-assistant/frontend#20444] [Configuration]
On large screens (>1600px width) there is more than enough space to leave the filter sidebar permanently open. This would make the filters easier to use and would also make them more visible to new users.
<img width="1914" alt="image" src="https://github.com/home-assistant/frontend/assets/30938717/c6607f9e-7aae-423e-a8bb-2536d7e92ca0">
Alternatively, it would be good if the status could be saved as it is with the main sidebar.

**Comment by ildar170975** [comment]
I would prefer a "pin" button.
Some users may not need the filter panel open permanently.
